vendors/laiier/vendor.yaml

```
@@ -1,21 +1,19 @@
 # ID of the vendor to be used as an internal reference for Actility vendors.
-# Name of vendor is in lowercase with a maximum of 8 characters.
+# Define your own ID following this rule: Maximum of 8 characters in lowercase.
+# Be careful: Once you chose a vendor ID, it cannot be changed later
 id: laiier
-# Formal/commercial name of the vendor to be used as a label.
-name: Laiier
-# Simple description of the vendor.
-description: "We give you the tools to make any building or facility smarter and better protect your assets. Making a building more efficient, sustainable, and smarter starts with detailed knowledge of what is happening within and how to take action."
-# LoRa-Alliance Vendor ID used for QR code.
-# It should be in hexadecimal format (2 octets). example: 000E
-# Reference url: https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf
-loRaAllianceVendorID: 
+# Official/commercial name of the vendor to be used as a label.
+name: LAIIER
+# Simple description of the vendor. Maximum 500 characters. Special characters are not allowed ('\n', '\t', '\r', ect...).
+description: LAIIER gives you the tools to make any building or facility smarter and better protect your assets.
+
+# LoRa-Alliance Vendor ID used for QR code https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf. It consists of 4 upper-case hexadecimal characters.
+loRaAllianceVendorID: 0000
 # Logo file name if different from <vendorId>.png (optional).
-logo: laiier-logo.png
+logo: LAIIER-logo.png
```

lower case for logo name
```suggestion
logo: laiier-logo.png
```